src/components/Camera.tsx

```diff
@@ -42,34 +58,26 @@ export const Camera = () => {
             modelType: "full",
           };
 
-        const detector = await handPoseDetection.createDetector(
+        detector = await handPoseDetection.createDetector(
```

Comment:
useEffect may be cleaned up during any await. Check for termination should be after this await too.